app/src/main/res/layout/activity_folder_view.xml

```diff
@@ -1,113 +1,116 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
This looks like a big diff, but it only adds the swiperefresh. Everything else gets moved down one level and indented; nothing else changed.